tests/utils.py

```diff
@@ -18,6 +18,13 @@
 packageName = 'ga4gh'
 
 
+def zipSameLength(*iterables):
+    length = len(iterables[0])
+    for iterable in iterables[1:]:
+        assert len(iterable) == length
```

Comment:
I think that's fine.  We're not dealing with large enough sequences that we can't call `list(generator)` beforehand anyways.  In fact, we often do just this in tests.